examples/openfaas/python/echo/stack.yml

```diff
@@ -1,7 +1,7 @@
 version: 1.0
```

Comment:
nit: obsolet

Reply:
Gosh - had no idea I'd committed that! Will remove.

Reply:
Note that this is actually me accidentally push my local changes to the OpenFaaS echo function. I've reverted these changes to the upstream/development version, so this should be ignored when I squash I think.